src/Model/Resource.php

```diff
@@ -4,8 +4,9 @@
 
 use Cake\Datasource\EntityInterface;
 use Cake\Datasource\EntityTrait;
+use Cake\Datasource\InvalidPropertyInterface;
 
-class Resource implements EntityInterface
+class Resource implements EntityInterface, InvalidPropertyInterface
```

Comment:
Have to remove this, this makes it dependent on CakePHP 3.2. We'll have to add this once we tag the plugin for 3.1.